checker/examples/units-extension/qual/FrequencyRelations.java

```diff
@@ -34,7 +34,8 @@ public UnitsRelations init(ProcessingEnvironment env) {
     }
 
     /** No multiplications yield Hertz. */
-    public /*@Nullable*/ AnnotationMirror multiplication(AnnotatedTypeMirror lht, AnnotatedTypeMirror rht) {
+    public /*@Nullable*/ AnnotationMirror multiplication(
```

Comment:
I think ant reformat changed the format for some reason.
I'll look into why that happened and let you know.